ntoskrnl/lpc/port.c

```diff
@@ -287,8 +287,40 @@ NtQueryInformationPort(IN HANDLE PortHandle,
                        IN ULONG PortInformationLength,
                        OUT PULONG ReturnLength)
 {
-    UNIMPLEMENTED;
-    return STATUS_NOT_IMPLEMENTED;
+	KPROCESSOR_MODE PreviousMode;
```

Comment:
Please fix the whole code style at once (including the 4-space indentation) in this commit (+squash together with the next commit).